### ISSUE TITLE
Current Twitter api link doesn't work

### ISSUE THREAD

**Issue by subdesign** [joeauty/TwitterFetcher-CodeIgniter-Spark#1]
Replacing the line 116 and 119 in twitterfetcher.php with the lines below:

116: $tweeturl = 'statuses/user_timeline.' .$configObj['format']. '?screen_name=' .$configObj['twitterID']. '&count' . $configObj['count'];

119: $tweeturl = 'statuses/user_timeline.' .$configObj['format']. '?screen_name=' .$configObj['twitterID'];

makes the script work well.

**Comment by joeauty** [comment]
I'll test these changes ASAP, but for both now and in general feel free to fork and pull request. I appreciate your interest in making this better and I most definitely will commit your changes promptly! :)

**Comment by joeauty** [comment]
I forgot to add that the whole reason behind the cache is because I suspect that Twitter rate limits, and so the cache will prevent having the Twitter servers queried each page load, limiting hitting rate limit thresholds, if they do exist.

Could the problem you have described relate to these rate limits?

**Comment by subdesign** [comment]
I don't think so, in my case. I just played with the twitter console, and observe that the user_timeline part has different syntax, I changed to that, and it works now. Second, I don't think I hit the rate limit.

**Comment by joeauty** [comment]
You're absolutely right about this, I don't know how or why I came up with the URL scheme I came up with before, but the one in your modification is indeed the URL as documented in the Twitter API. As you no doubt know, Twitter does indeed rate limit, so perhaps pushing the cache feature a little more might be a good thing.

I also don't get into authentication, using the https domain, and other stuff - maybe this sort of thing would be a good addition for a future version of this Spark. 

I've version bumped the Spark to 1.0.3 which includes your patch: https://github.com/joeauty/TwitterFetcher-CodeIgniter-Spark/commit/e791c1841d975508a14d85b4572f648d49420806

http://getsparks.org/packages/twitterfetcher/show

Thanks again for your help!
